### PR TITLE
Vagrant + Packer improvements

### PR DESCRIPTION
**Pull request by josephglanville**
Few changes to get up and running using Packer + VMWare Fusion.

* Ubuntu 14.04.3 has been released, as such Ubuntu 14.04.2 is no longer available on release mirrors.
* Ignore the Packer build directories
* Configure the VMWare Fusion provider in Vagrantfile to match Virtualbox

### CODE REVIEW

**Comment by josephglanville** [comment]
Unfortunately using local builds the box URL and version constraint causes issues so you still need to edit the Vagrantfile to get a running setup, this isn't so bad though.

Perchance a compatible box could be published?

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ bin
 script/custom-vagrant
 demo/*.log
 util/_toolchain/**
+util/packer/ubuntu-14.04/output-*
```
Comment (lmars) [inline .gitignore]: Put this in `util/packer/.gitignore`

**Comment by lmars** [comment]
@josephglanville please add component prefixes to your commit messages (i.e. `Vagrantfile: xxx` or `util/packer: xxx`)

**Comment by lmars** [comment]
@josephglanville 

> using local builds the box URL and version constraint causes issues so you still need to edit the Vagrantfile to get a running setup

Can you describe the issues?

**Comment by josephglanville** [comment]
I will fix those up.

As for additional details, if you build a local VMware box using Packer and then import it as flynn-base, the version and box URL will not play ball. Publishing the VMware image would fix this, opening an issue to discuss options there.

**Comment by lmars** [comment]
Thanks!

**Pull request merged**